docker-for-windows/wsl-tech-preview.md

```diff
@@ -0,0 +1,54 @@
+---
+description: Docker Desktop WSL 2 Tech Preview
+keywords: Docker, WSL, WSL 2, Tech Preview, Windows Subsystem for Linux
+title: Docker Desktop WSL 2 Tech Preview
+toc_min: 1
+toc_max: 2
+---
+
+# Overview
+
+Welcome to Docker Desktop WSL 2 Tech Preview. This Tech Preview introduces support to run Docker Desktop with WSL 2. We really appreciate you trialing this Tech Preview. Your feedback is very important to us. Please let us know your feedback by creating an issue in the [Docker Desktop for Windows GitHub](https://github.com/docker/for-win/issues) repository and adding the **WSL 2** label.
```

Comment:
Do users know what WSL is?

Reply:
nice catch on WSL. I also asked Usha about that but because she's not back online tomorrow, we'll publish as is today and hopefully she'll make the fix to the topic tomorrow.